l10n_br_sale/sale.py

```diff
@@ -297,6 +306,12 @@ def _prepare_invoice(self, cr, uid, order, lines, context=None):
 class SaleOrderLine(orm.Model):
     _inherit = 'sale.order.line'
 
+    def _calc_line_base_price(self, cr, uid, line, context=None):
```

Comment:
Is it necessary to redefine those two methods? They are equal to the base sale.py.